web/themes/custom/tube_theme/scss/misc.scss

```diff
@@ -0,0 +1,45 @@
+// Recently Added Videos View
+
+.recent {
+  background-color: $bg-color;
+}
+.recent-video-view {
+  display: flex;
+  width: 100%;
+  justify-content: space-around;
+  height: 394px;
+  margin-top: 25px;
+}
+.block__title {
+    @include font-size(36px, 41px);
+    @include heading-font(500);
+    color: $black;
+    text-align: left;
+    margin-left: 48px;
+  }
+.view-name {
+  a {
+    @include open-sans-font(700);
+    @include font-size(18px, 25px);
+    color: $black;
+    text-align: left;
+    text-transform: none;
+  }
+}
+.speaker {
+  @include open-sans-font(600);
+  @include font-size(14px, 19px);
+  text-align: left;
+  text-transform: uppercase;
+  color: $light-gray;
+}
+.row {
+  flex-wrap: nowrap;
+}
+.container-fluid {
```

Comment:
The row and container-fluid classes are core parts of the bootstrap framework, we shouldn't change how they fundamentally work. I'll merge this PR but undo that change